.changeset/sweet-crabs-smile.md

```diff
@@ -0,0 +1,15 @@
+---
+"@wso2is/react-components": patch
+"@wso2is/access-control": patch
+"@wso2is/dynamic-forms": patch
+"@wso2is/identity-apps-core": patch
+"@wso2is/myaccount": patch
+"@wso2is/common": patch
+"@wso2is/theme": patch
+"@wso2is/console": patch
+"@wso2is/core": patch
+"@wso2is/form": patch
+"@wso2is/i18n": patch
```

Comment:
We should only select the packages updated with this PR. Please generate the changeset again only with the changed packages. :)

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4460/commits/1e515a6da610b6955d5d3b0ea4cc5413e7e640b3